js/packages/phoenix-client/src/types/experiments.ts

```diff
@@ -0,0 +1,78 @@
+import { AnnotatorKind } from "./annotations";
+import { Node } from "./core";
+import { Example } from "./datasets";
+
+/**
+ * An experiment is a set of task runs on a dataset version
+ */
+export interface Experiment extends Node {
+  id: string;
+  datasetId: string;
+  datasetVersionId: string;
+  repetitions: number;
+  /**
+   * The project under which the experiment task traces are recorded
+   */
+  projectName: string;
+}
+
+/**
+ * The result of running an experiment on a single example
+ */
+export interface ExperimentRun extends Node {
+  id: string;
+  startTime: Date;
+  endTime: Date;
+  /**
+   * What experiment the run belongs to
+   */
+  experimentId: string;
+  datasetExampleId: string;
+  repetitionNumber: number;
+  output?: string | {} | null;
```

Comment:
I don't think this is right {} is like an empty object i think maybe:

```suggestion
  output?: string | Record<string, unknown> | null;
```
or more defensively
```suggestion
  output?: string | Record<unknown, unknown> | null;
```

Reply:
It's technically anything that can be JSON dumped. I'll commit the former for now